index.html

```diff
@@ -3,7 +3,11 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Calculator</title>
+    <link href="css/index.css" rel="stylesheet" />
   </head>
-  <body></body>
+  <body>
+    <h1>hello world</h1>
+    <script src="js/index.js"></script>
```

Comment:
Also i suggest you remove this for now and focus on the purpose of the PR ,you can implement that when js is needed for functionality, just like you'll create the utility class list file when you're about to work on design